test/tests/ver-1ja/tests

```diff
@@ -0,0 +1,18 @@
+[Tests]
+  design = 'ver-1ja.md'
+  verification = 'ver-1ja.md'
+  issues = '#145'
+  [ver-1ja_csvdiff]
+    type = CSVDiff
+    input = ver-1ja.i
+    csvdiff = ver-1ja_out.csv
+    requirement = 'The system shall be able to model decay of tritium and associated growth of He in a diffusion segment, with the fine mesh and timestep required to match the analytical solution to generate CSV data for use in comparisons with the analytic solution.'
```

Comment:
Do we need the fine mesh and timestep requirement part if the test isn't heavy?

Reply:
I removed it.